src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,432 @@
+				dialog.setVisible(false);
+			}
+		});
+
+		dialog.setSize(600, 250);
+		dialog.setLocationRelativeTo(mcreator);
+		dialog.setVisible(true);
+	}
+
+	private static void addWoodPackToWorkspace(MCreator mcreator, Workspace workspace, String name,
+			Color color, double factor) {
+		// first we generate wood texture
+		ImageIcon wood = ImageUtils.colorize(ImageMakerTexturesCache.CACHE.get(new ResourcePointer(
+						"templates/textures/texturemaker/" + ListUtils
+								.getRandomItem(Arrays.asList("log_side_1", "log_side_2", "log_side_3",
+										"log_side_4", "log_side_5")) + ".png")), color,
+				true);
+		String woodTextureName = (name + "_log_side").toLowerCase(Locale.ENGLISH);
+		FileIO.writeImageToPNGFile(ImageUtils.toBufferedImage(wood.getImage()),
+				mcreator.getWorkspace().getFolderManager()
+						.getBlockTextureFile(RegistryNameFixer.fix(woodTextureName)));
+
+		//then we generate the missing log texture
+		ImageIcon log = ImageUtils.colorize(ImageMakerTexturesCache.CACHE.get(new ResourcePointer(
+						"templates/textures/texturemaker/log_top.png")), color, true);
+		String logTextureName = (name + "_log_top").toLowerCase(Locale.ENGLISH);
+		FileIO.writeImageToPNGFile(ImageUtils.toBufferedImage(log.getImage()),
+				mcreator.getWorkspace().getFolderManager()
+						.getBlockTextureFile(RegistryNameFixer.fix(logTextureName)));
+
+		//then we generate the planks texture
+		ImageIcon planks = ImageUtils.colorize(ImageMakerTexturesCache.CACHE.get(new ResourcePointer(
+						"templates/textures/texturemaker/" + ListUtils
+								.getRandomItem(Arrays.asList("planks_0", "planks_1")) + ".png")), color,
+				true);
+		String planksTextureName = (name + "_planks").toLowerCase(Locale.ENGLISH);
+		FileIO.writeImageToPNGFile(ImageUtils.toBufferedImage(planks.getImage()),
+				mcreator.getWorkspace().getFolderManager()
+						.getBlockTextureFile(RegistryNameFixer.fix(planksTextureName)));
+
+		//then we generate the leaves texture
+		ImageIcon leaves = ImageUtils.colorize(ImageMakerTexturesCache.CACHE.get(new ResourcePointer(
+						"templates/textures/texturemaker/" + ListUtils
+								.getRandomItem(Arrays.asList("leaves_0", "leaves_1", "leaves_2", "leaves_3", "leaves_4",
+										"leaves_5", "leaves_new1", "leaves_new2", "leaves2")) + ".png")), color,
+				true);
+		String leavesTextureName = (name + "_leaves").toLowerCase(Locale.ENGLISH);
+		FileIO.writeImageToPNGFile(ImageUtils.toBufferedImage(leaves.getImage()),
+				mcreator.getWorkspace().getFolderManager()
+						.getBlockTextureFile(RegistryNameFixer.fix(leavesTextureName)));
+
+		// we use Block GUI to get default values for the block element (kinda hacky!)
+		Block woodBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "Wood", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		woodBlock.name = name + " Wood";
+		woodBlock.material = new Material(workspace, "Wood");

```

Comment:
Material is not set because there isn't a wood material capitalized like that. Please use "WOOD" instead.